src/zm_db.cpp

```diff
@@ -37,23 +37,35 @@ void zmDbConnect()
   my_bool reconnect = 1;
   if ( mysql_options( &dbconn, MYSQL_OPT_RECONNECT, &reconnect ) )
     Fatal( "Can't set database auto reconnect option: %s", mysql_error( &dbconn ) );
-  std::string::size_type colonIndex = staticConfig.DB_HOST.find( ":/" );
-  if ( colonIndex != std::string::npos )
+  std::string::size_type colonIndex = staticConfig.DB_HOST.find( ":" );
```

Comment:
I think there is a problem in here.  So, the new logic is, if we found a colon, then we look for a socket path?  That will never happen. The old code looked for either a colon or a slash, and then looked for a /. (By old code I mean some other patch that I merged into storageareas).